### DISCUSSION TITLE
Highlight countries on hover

### DISCUSSION THREAD

**Discussion by zacBkh** [uber/react-map-gl#2234] [Q&A]
Hello,
I can't wrap my head around on how can I implement a map where when the user hover over it, the hovered country is highlighted. 
For example, its fill color would change. 

Do I need to create a custom tileset? Does mapbox provide a solution out of the box?

Thanks!

**Comment by Pessimistress** [comment]
https://visgl.github.io/react-map-gl/examples/filter

**Comment by zacBkh** [comment]
> https://visgl.github.io/react-map-gl/examples/filter

Thanks a lot!

I just have few questions:

- What is this : mapbox://mapbox.82pkq93d ?
Is it a custom tileset? If I understand well it's on this tileset that the counties information are registered and that the feature property is populated on map hover ?

- When specifying interactiveLayerIds, in the array you put "counties". Is it referring to the layer you have on your tileset ?


I already have a tileset in mapbox studio where each countries has its own information etc but I'm struggling on how to import these information on my actual <Map/> on my front end
